cliclient/cliclient.go

```diff
@@ -99,8 +126,31 @@ func NewProjectClient(config *config.ServerConfig) (*MasterClient, error) {
 	return mc, nil
 }
 
+var testingForceClientInsecure = false
+
+/*
+TestingForceClientInsecure sets testForceClientInsecure to true for unit tests.
+It's a workaround to github.com/rancher/norman/clientbase.NewAPIClient,
+which replaces net/http.Client.Transport (including proxy and TLS config),
+so the client TLS config of net/http/httptest.Server will be lost.
+Not thread-safe.
+Call the returned function by defer keyword, for example:
+
+	defer cliclient.TestingForceClientInsecure()()
+*/
+func TestingForceClientInsecure() func() {
```

Comment:
Can we move it undert the `cliclient_test.go` file?

Reply:
Same situation to `TestingReplaceDefaultHTTPClient`.